.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -145,6 +145,8 @@ AddType text/vtt                            vtt
     RewriteRule ^acs/installing/install-ocp-operator.html /acs/4.0/installing/installing_ocp/init-bundle-ocp.html [NE,R=301]
     RewriteRule ^acs/(\D.*)$ /acs/4.0/$1 [NE,R=301]
     RewriteRule ^acs/(3\.65|3\.66|3\.67|3\.68|3\.69|3\.70|3\.71|3\.72|3\.73|3\.74|4\.0)/?$ /acs/$1/welcome/index.html [L,R=301]
+    #redirect for 4.0 Manage vulneribility page
+    RewriteRule ^(acs/(?:\d\.\d/)?)?operating/manage-vulnerabilities\.html$ /acs/4.0/operating/manage-vulnerabilities/vulnerability-management.html [NE,R=301]
```

Comment:
```suggestion
    RewriteRule ^(acs/(?:\d\.\d+/)?)?operating/manage-vulnerabilities\.html$ /acs/4.0/operating/manage-vulnerabilities/vulnerability-management.html [NE,R=301]
```
As written, this rule will only catch version numbers like "3.7" but not "3.74". The `+` should let it catch both types of version number. Feel free to run it through https://htaccess.madewithlove.com/ to test.

Reply:
Thank you @bscott-rh